packages/inter-protocol/src/auction/scheduler.js

```diff
@@ -184,21 +185,31 @@ export const makeScheduler = async (
         wake(time) {
           setTimeMonotonically(time);
           trace('wake step', now);
-          clockTick(liveSchedule);
+          try {
+            clockTick(liveSchedule);
+          } catch (err) {
+            console.error('🚨 clockTick failed:', err);
+          }
         },
       }),
       stepCancelToken,
     );
   };
 
   // schedule a wakeup for the next round
+  let lastStartNextRound;
   const scheduleNextRound = start => {
     trace(`nextRound`, start);
+    if (lastStartNextRound === start) {
+      console.error('scheduleNextRound called with same start', start);
+    }
+    lastStartNextRound = start;
     void E(timer).setWakeup(
       start,
       Far('SchedulerWaker', {
         wake(time) {
           setTimeMonotonically(time);
+          // ??? if wake() returns a rejection, what happens? should we return startAuction()?
```

Comment:
FYI vat-timer will log any rejections it gets from `wake()`, as `console.log('one-shot event error', err)` from the timer vat (not the most obvious place to look for error messages, but better than nothing). Since this is a `setWakeup`, not a repeater, there's nothing on the vat-timer side to cancel or to decide to not reschedule.